IpmiFeaturePkg/GenericIpmi/Pei/PeiGenericIpmi.c

```diff
@@ -54,85 +54,125 @@ PeimIpmiInterfaceInit (
   IPMI_BMC_HOB            *BmcHob;
 
   mIpmiInstance = NULL;

```

Comment:
Not related to your change necessarily, but why do these local variables use the module level variable naming convention (prefixed with `m`)?

Reply:
Pushing a change to drop the m prefixes for the Pei.

Reply:
fixed in latest